debezium-embedded/src/main/java/io/debezium/embedded/ConvertingEngineBuilder.java

```diff
@@ -197,13 +206,23 @@ private Converter createConverter(Class<? extends SerializationFormat<?>> format
         converterConfig = commonConverterConfig.edit().with(converterConfig).build();
 
         if (isFormat(format, Json.class)) {
-            converterConfig = converterConfig.edit().withDefault(FIELD_CLASS, "org.apache.kafka.connect.json.JsonConverter").build();
+            if (converterConfig.hasKey(APICURIO_SCHEMA_REGISTRY_URL_CONFIG)) {
```

Comment:
This is good for Avro but I am not conviced we should follow the same logic for Json converter too.

I'd probably either require explicit setting of the converter or add a config option would switch to apicurio's.

@gunnarmorling Any thoughts on that?

Reply:
So I think if a user specifies a schema registry URL, it's fair to assume that they also mean to use it. This shouldn't need yet another option. Let's clarify the behavior in the docs, though.

Reply:
@ani-sha I've just synced with Gunnar and we decided it should stay at is right now. No need for additional option.
This behaviour just needs to be documented.